Improve map rendering

godsic opened the pull request:
by making Cesium fetch higher resolution tiles more aggressively as
suggested in https://github.com/CesiumGS/cesium/issues/3279#issuecomment-271481280

![image](https://user-images.githubusercontent.com/2339291/179087261-8720b5c8-e0fb-4d2a-9bed-26bb603e1d1c.png)

godsic committed:
Improve map rendering

by making Cesium fetch higher resolution tiles more aggressively as
suggested in
https://github.com/CesiumGS/cesium/issues/3279#issuecomment-271481280

ckoval7 commented:
A value of 1.0 causes a noticeable hit in performance. Perhaps the recommended value of 1.33 would be better as recommended in the issue you linked, as well as a related issue.

godsic commented:
@ckoval7 Aha, I have not noticed any regressions on my average Intel NUC, but perhaps lower power hardware might straggle a bit indeed. Perhaps a better idea is to add checkbox that toggles higher quality map rendering as I did in my latest commit?

godsic commented:
@ckoval7 I might forgot to mention, but when I force pushed I added the button I've mentioned. My bet, should not force push ever) Anyway, that's how it looks like:
![image](https://user-images.githubusercontent.com/2339291/182714775-d86beaa7-631d-4ec9-bb66-a98e43698fea.png)

ckoval7 commented:
I am going to implement this a little differently. It'll be entirely in the browser, not having to touch the python side of things.

godsic commented:
Sure, I am not an expert in web development at all, just mimicked how other buttons were implemented) does not make sense to have a state for such thing on the server side.

alphafox02 commented:
With a little extra leg work, it's possible to run offline maps w/ DFA. I just need a little help to make the process easier. Here's the current process, just replace PhotonMap w/ DFA. 

https://www.youtube.com/watch?v=dZu03Wcisv4

godsic committed:
Bump CesiumJS to 1.104

godsic committed:
Don't redraw scene for no reason

godsic committed:
Don't reset and disable reciever if it fails

Instead keep the last know measurement and plot corresponding lob in
gray color to alert operator. This change is suggested to be used
together with https://github.com/krakenrf/krakensdr_doa/pull/72 .

godsic committed:
Add label with station ID for convenience

godsic committed:
Remove car and tower icons

godsic committed:
Add description to the station label

The description could be viewed in InfoBox for operator's convenience.

godsic committed:
Improve UI accessibility on small screens

godsic committed:
Act consistently if connection to reciever is lost

godsic committed:
Adapt update rate to DoA processing time

Also redraw map every 1s instead of 2.5s.

godsic committed:
Extend station information provided in infoBox

godsic committed:
Show lob in red if DoA estimation is unreliable

Which includes ADC overdrive, low SNR or signal correlation.

IIvanIvanovich commented:
Hi! Will these changes be included in the main branch? I want to try them at work.

ckoval7 commented:
The code base has changed substantially. Open targeted issues for Cesium render-mode flags and receiver-info-box telemetry, which are the parts of this still worth pursuing.
